use_case_examples/cifar/README.md

```diff
@@ -0,0 +1,59 @@
+# CIFAR-10 and CIFAR-100 Classification with FHE
+
+This repository provides resources and documentation on different use-cases for classifying CIFAR-10 and CIFAR-100 images using Fully Homomorphic Encryption (FHE). Each use-case demonstrates different techniques and adaptations to work within the constraints of FHE.Notably, a fine-tuning from a public pre-trained model, a training from scratch using quantization aware training (QAT) and finally a hybrid approach where only a subset of the model is done in FHE.
```

Comment:
there's a small typo : `FHE.Notably` -> FHE. Notably

Reply:
yes it's one of my tic de langage